base/iterators.jl

```diff
@@ -32,7 +32,7 @@ import Base:
     getindex, setindex!, get, iterate,
     popfirst!, isdone, peek, intersect
 
-export enumerate, zip, rest, countfrom, take, drop, takewhile, dropwhile, cycle, repeated, product, flatten, flatmap, partition
+export enumerate, zip, rest, countfrom, take, drop, takewhile, dropwhile, cycle, repeated, product, flatten, flatmap, partition, findall
```

Comment:
All these exports are bad because they conflict with Base exports. There's no use (unless you use `baremodule`). Let's not add another export (make it public :) )

Reply:
(if the name is changes to findeach, this comment is not applicable)